src/utils/config.py

```diff
@@ -14,6 +14,7 @@ class VTRConfig:
     hidden_size_ocr: int
     num_layers_ocr: int
     ratio: float
+    char2array: str
```

Comment:
See `tokenizer` argument in `configure_arg_parser` in `main.py`. Let's do the same.

Reply:
Did I understand correctly that I should move `char2array` parameter from `VTRConfig` to `main` arg parser?

Reply:
Yes!